### ISSUE TITLE
Bitmap not drawing when created before window

### ISSUE THREAD

**Issue by Ian-RG** [splashkit/splashkit-core#97]
I've come across what appears to be a bug. When running this code, the bitmap user_image will not draw to the window if the window is initialised after the bitmap. When the line initialising the window is moved above the initialisation of the bitmap, the code runs normally.

```
int main()
{
    bitmap user_image;

    user_image = create_bitmap ("user_image", WIDTH, HEIGHT);
    clear_bitmap (user_image, COLOR_WHITE);
   
    window my_window = open_window ("Image Editor", WIDTH, HEIGHT);        
    clear_window (my_window, COLOR_WHITE);   
    fill_rectangle_on_bitmap (user_image, COLOR_BLACK, 0, 0, 100, 100);      

    draw_bitmap_on_window (my_window, user_image, 0, 0);
    refresh_window(my_window);
    delay (5000);

    
    return 0;
}
```

**Comment by macite** [comment]
I have a solution to this working on Mac, and tested on Windows. Just need to test on Linux then deploy.

**Comment by macite** [comment]
Fixed in commit 6438ce31c6d56e8dc03dc1112192ed690452777e
